### PR TITLE
Update the testing document with latest content from TEST.md

### PR DESCRIPTION
**Pull request by willbarton**
The manual didn't have the latest content moved over from TEST.md. This PR fixes that.
 
## Changes

- Updates docs/testing.md with the last good content from TEST.md

## Review

- @richaagarwal

### CODE REVIEW

**Comment by richaagarwal** [comment]
Perf thanks! 👍

File: docs/testing.md [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
 
-Open a new Terminal window or tab and change to the project directory,
+Many of our browser tests have been rewritten as Python unit tests. The remaining rely on data that is not generated during testing, so please note many of these tests might fail if you do not have a production database locally. If you do, open a new Terminal window or tab and change to the project directory,
 then tell gulp to start the tests:
 
 ```sh
 gulp build
-gulp test:acceptance
+gulp test:acceptance --suite=integration
+gulp test:acceptance —suite=content
```
Comment (anselmbradford) [inline docs/testing.md]: Looks like argument is using an m-dash instead of two n-dashes

File: docs/testing.md [hunk 2]
```diff
@@ -47,15 +49,17 @@ Sauce Labs can be used to run tests remotely in the cloud.
 
 7. Run the tests with `gulp test:acceptance`.
 
-    > Note: If you want to temporarily disable testing on Sauce Labs,
-    run the command as: `gulp test:acceptance --sauce=false`.
+    !!! Note:
```
Comment (anselmbradford) [inline docs/testing.md]: TRIVIAL: In the past we used `>` for notes to separate and highlight them from the main body content, e.g.:

```
> NOTE:
   Blah, blah, blah
```

Comment (anselmbradford) [inline docs/testing.md]: Oh, wait, that's what got deleted, haha

But yeah, the reason for that was to offset the notes from regular paragraphs.

Comment (willbarton) [inline docs/testing.md]: The `!!! Note` creates a callout when formatted by mkdocs, like this:

image

File: docs/testing.md (hunk 1 above)
Comment (willbarton) [inline docs/testing.md]: Good catch! Fixed.

**Comment by contolini** [comment]
Baller. :shipit:

File: docs/testing.md (hunk 2 above)
Comment (anselmbradford) [inline docs/testing.md]: Ah, cool! Makes sense

**Pull request merged**